src/components/Header/components/NavButtonsRightCorner/index.jsx

```diff
@@ -7,6 +7,7 @@ import { useAuthGroups, useNetworkStatus } from 'hooks';
 import { useState } from 'react';
 import { AuthGroup, signOut } from 'services/auth';
 import styles from './NavButtonsRightCorner.module.css';
+import { ModeSwitcher } from 'components/ThemeModeSwitcher';
```

Comment:
Make sure to remove this before we merge, we don't want to leave the test button in!